pkgs/by-name/me/mealie/package.nix

```diff
@@ -0,0 +1,165 @@
+{ lib
+, callPackage
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, nixosTests
+, python3
+, stdenv
+, writeShellScript
+}:
+
+let
+  version = "1.2.0";
+  src = fetchFromGitHub {
+    owner = "mealie-recipes";
+    repo = "mealie";
+    rev = "v${version}";
+    sha256 = "sha256-Kc49XDWcZLeJaYgiAO2/mHeVSOLMeiPr3U32e0IYfdU=";
+  };
+
+  frontend = callPackage (import ./mealie-frontend.nix src version) { };
+
+  python = python3.override {
+    packageOverrides = self: super: {
+      pydantic = self.pydantic_1;
+    };
+  };
+
+  crfpp = stdenv.mkDerivation {
+    pname = "mealie-crfpp";
+    version = "master";
```

Comment:
This should be in the unstable-XXXX-XX-XX where the X's are the date

Reply:
I'm sorry I don't understand, I should add `crfpp` to the whole nixpkgs packages ?
It's something fairly unmaintained, and here forked specifically for `mealie`, I don't really know if that's a wise choice

Reply:
If this is a package others might rely on, yes. If this is a "vendored" dependency, it's fine to keep it in here. Though as @SuperSandro2000 said, please use the standard version format used for things built from git revisions rather than tags.

Reply:
Ok I see, fixed